tests/plugins/balloontoolbar/plugin.js

```diff
@@ -0,0 +1,26 @@
+/* bender-tags: balloontoolbar, 1346, context */
```

Comment:
`plugin.js` is not very descriptive name for test suite, maybe something like `contextinit.js` (well, it will be nice if the naming will be consistent with manual test name, so you could go with `contextapi`)?